### PR TITLE
Add nb_NO translation from Weblate

### PR DESCRIPTION
**Pull request by jayvdb**
Proof of concept for https://github.com/coala/gci-leaders/issues/124

### CODE REVIEW

**Comment by jayvdb** [comment]
Travis doesnt like the commit message:

```
 Translated using Weblate (Norwegian Bokmål)

Currently translated at 100.0% (19 of 19 strings)

Translation: Coala/GCI Leaders
Translate-URL: hosted.weblate.org/projects/coala/gci-leaders/nb_NO
```

> Shortlog of HEAD commit isn't in imperative mood! Bad words are 'Translated'
> Body of HEAD commit contains too long lines. Commit body lines should not exceed 72 characters.


I've updated the template shortlog, so this shouldnt occur again

**Review by blazeu — APPROVED** [review]

File: static/js/i18n/nb_NO.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "page-title": "Nåværende",
```
Comment (andrewda) [inline static/js/i18n/nb_NO.json]: In the other i18n translations, 2 space indentations are used. I don't mind either way, but we should keep it consistent.

**Review by andrewda — CHANGES_REQUESTED** [review]

**Review by realslimshanky — CHANGES_REQUESTED** [review]
Please rebase the branch of your fork with the master of this repository.

File: static/js/i18n/nb_NO.json (hunk 1 above)
Comment (jayvdb) [reply static/js/i18n/nb_NO.json]: Interesting problem.  These files come from Weblate, and its generator is not configurable as far as I can see.
So, I think we need to change our data files to match the Weblate ones.

**Comment by blazeu** [comment]
ack https://github.com/coala/gci-leaders/pull/107/commits/9a170ecf6e39813f0a722fee6a5ba8378b9af3cb https://github.com/coala/gci-leaders/pull/107/commits/52678294a5f9db4684e4db2f1ae23495d3016c54 https://github.com/coala/gci-leaders/pull/107/commits/572c9bf0846e6729dc0b79908f4fec5bc95e435b https://github.com/coala/gci-leaders/pull/107/commits/d29de0a23dcf349ae2877911ad016a2236c2e86e

**Pull request merged**